cli/cmd/timescaledbGetPassword.go

```diff
@@ -19,7 +19,7 @@ var timescaledbGetPasswordCmd = &cobra.Command{
 
 func init() {
 	timescaledbCmd.AddCommand(timescaledbGetPasswordCmd)
-	timescaledbGetPasswordCmd.Flags().StringP("user", "U", "PATRONI_SUPERUSER_PASSWORD", "user whose password to get")
+	timescaledbGetPasswordCmd.Flags().StringP("user", "U", "", "user whose password to get, if not provided will be used super-user used for deployment.")
```

Comment:
`, if not provided will be used super-user used for deployment.` is used several times where a `user` flag is defined for a command, but:
1. can be reworded as `if not provided super-user will be used for deployment`
2. we're not always doing a deployment in some of these commands, we need a diff msg there (i.e. here)